python/ray/tests/test_actor.py

```diff
@@ -1431,6 +1431,22 @@ def ping(self):
     assert ray.get(detached_actor.ping.remote()) == "pong"
 
 
+def test_force_kill(ray_start_regular):
+    @ray.remote
+    class Actor(object):
+        def hang(self):
+            # Never returns.
+            ray.get(ray.ObjectID.from_random())
+
+    actor = Actor.remote()
+    result = actor.hang.remote()
+    ready, _ = ray.wait([result], timeout=0.1)
+    assert len(ready) == 0
+    actor.force_kill()
```

Comment:
I don't find where you inject this `force_kill` into the actor handle object. Did you forget that?

Reply:
Ah sorry I renamed this but didn't update the test. It's now `__ray_kill__`